queries/rbs/highlights.scm

```diff
@@ -115,6 +130,10 @@
 (type
   (integer_literal) @number)
 
+(type
+  (record_type
+    key: (record_key) @string.special.symbol))
```

Comment:
if this is a key in a key/value pair, it should be `@property`

Reply:
in RBS, the key means the Hash table's key in Ruby language. The key class is Symbol.
I think `@string.special.symbol` represents the key semantics.

Reply:
Do you have an example for me?

Reply:
I show a example.

foo.rbs: this is a type signature file.
```rbs
class Foo
  @record: {a: String, b: Integer}
end
```

foo.rb: this is an actual Ruby code.
```ruby
class Foo
  def initialize
    @record = {a: "string value", b: 1234}
  end

  def get_a
    @record[:a]
  end
end
```

In this case, the record key in the rbs file means the fixed-key Hash table type in Ruby code.

In Ruby code, the Symbol class is used as a key to refer to this value, which is represented as `@string.special.symbol` in Ruby syntax highlighting.

Since they refer to the same thing semantically, it is easier for programmers to understand if they are highlighted the same way.